src/js/Keyboard.js

```diff
@@ -234,20 +227,41 @@ class Keyboard extends Component {
 
     var interactionText = app && app.interactionText && app.interactionText.fieldText ? 
       app.interactionText.fieldText : "Tap on the virtual keyboard to start";
+
+    var parsedInput = this.state.input.replaceAll("\t", "        ");
+    var calculatedInputSize = parsedInput ? parsedInput.length : interactionText.length;
+
+    var inputClassName = "div-input"
+    if (parsedInput.length && (this.maskedInput || this.state.userMaskedInput)) {
+      inputClassName += " text-security"
```

Comment:
Shouldn't the input be displayed to the user while using `LIST_WITH_SEARCH`/`ICON_WITH_SEARCH`. Any reason as to why we'd want to obfuscate the search string.

Reply:
So these changes were more for fixing the masked input when i switched from a input html field to a plain ol div. This pr is based of those changes. That brings up a good question on why a dev would ever want this, but thats kinda how the keyboard properties are supposed to work i think